libs/ledger-live-common/src/families/near/js-transaction.ts

```diff
@@ -30,5 +25,6 @@ export const prepareTransaction = async (a: Account, t: Transaction): Promise<Tr
       })
     : t.amount;
 
-  return { ...t, fees, amount };
+  const newTx = { ...t, fees, amount };
+  return isEqual(t, newTx) ? t : newTx;
```

Comment:
Is there a reason to not use the `defaultUpdateTransaction` here with `{ fees, amount }` as patch ?